presto-main/src/main/java/com/facebook/presto/operator/OperatorInfo.java

```diff
@@ -14,6 +14,7 @@
 package com.facebook.presto.operator;
 
 import com.facebook.presto.operator.PartitionedOutputOperator.PartitionedOutputInfo;
+import com.facebook.presto.operator.WindowOperator.WindowInfo;
```

Comment:
Commit message doesn't need to discuss explain analyze at all, they're independent.